neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import datajoint as dj
+
+
+from neuro_data.static_images.data_schemas import StaticScanCandidate, StaticScan, ConditionTier, Frame, InputResponse, Eye, Treadmill, StaticMultiDataset, StaticMultiDatasetGroupAssignment
```

Comment:
I normally import explicitly to only import what I need, rather then just everything.

Reply:
I was saying to use `from neuro_data.static_images import data_schemas` rather than `from neuro_data.static_images.data_schemas import StaticScanCandidate`